style.css

```diff
@@ -351,3 +351,97 @@ h2 {
   align-items: flex-start;
   gap: 15px;
 }
+
+.contact-form {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  background-image: url(assets/images/background2.png);
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  background-size: 70%;
+  background-position-x: right;
+  background-position-y: 20px;
+  border-radius: 90px 0 0 0;
+  padding-top: 103px;
+}
```

Comment:
- [ ] Kindly make sure the background image `assets/images/background2.png` you are targeting is stored in your images folder.

    - ### Your design
   ![image](https://user-images.githubusercontent.com/87197412/199469311-1b34e5b4-73fa-4bfa-93a8-8d92c09cfcfe.png)

  - ### Figma design
 
   ![image](https://user-images.githubusercontent.com/87197412/199469710-a44fdf89-92f5-4827-ad26-39036af17273.png)